lib/metalsmith-lunr-index/index.js

```diff
@@ -29,6 +29,38 @@ module.exports = function lunrPlugin () {
           }
         })
 
+    const pageHeadingDocuments =
```

Comment:
can filter to that specificity if required yeah

Reply:
Even if we don't have the side bar the inline nav is still functionally the same so I'd be keen to keep these in the search results...

Reply:
My thinking is that there's a risk putting all headings in the search will overwhelm the user, whereas we know the pages that are most likely to be complex, since we've added a subnavigation.

Let's see how it goes...

Reply:
Just noticed this only exposes headings for the styles page, so that makes it less risky.

Reply:
Yeah, it's only items that are in what we originally called "thematic pages", that contain things that could be their own page, but made sense to group. e.g. Headings and Lists within Typography. We wouldn't want to expend this any further as all of the headings on pages that are about one thing have the same sets of headings

Reply:
updated to only index pages with `show_subnav: true`